lib/reports/eligible_commitments.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require "services"
+require "overlap/holding_commitment"
+
+module Reports
+  # Given criteria, pull up all holdings that match those criteria
+  # AND are eligible for commitments AND have no commitments.
+  # Invoke via bin/reports/compile_eligible_commitments_report.rb
+  class EligibleCommitments
+    def header
+      ["organization", "oclc_sym", "ocn", "local_id"]
+    end
+
+    def for_ocns(ocns = [])
+      if ocns.empty?
+        raise "No ocns given"
+      end
+
+      ocns.sort.uniq.each do |ocn|
+        overlap = Overlap::HoldingCommitment.new(ocn)
```

Comment:
Consider merging what we need from `Overlap::HoldingCommitment` back to this class?

Reply:
I think we landed on maybe not, since other similar Qs about commitments will arise outside the scope of this report.

Reply:
:+1: